packages/discovery-provider/ddl/migrations/0000_schema_failable.sql

```diff
@@ -1508,6 +1508,7 @@ CREATE TABLE public.tracks (
     is_playlist_upload boolean DEFAULT false NOT NULL,
     duration integer DEFAULT 0,
     ai_attribution_user_id integer
+    parent_album_ids number[]
```

Comment:
this will need to be in a new migration file versus this one.

new number in https://github.com/AudiusProject/audius-protocol/tree/main/packages/discovery-provider/ddl/migrations and add the sql to ALTER TABLE ADD COLUMN there.

Here's an example https://github.com/AudiusProject/audius-protocol/blob/main/packages/discovery-provider/ddl/migrations/0037_add_delist_reasons.sql

Reply:
Here's the README, but it could improve probably! 
https://github.com/AudiusProject/audius-protocol/tree/main/packages/discovery-provider/ddl

Reply:
Ahh yes that makes total sense. I remember learning about this way back